### ISSUE TITLE
Troubleshooting the "Run a Model" section of the readme

### ISSUE THREAD

**Issue by Jsevillamol** [ctlearn-project/ctlearn#49]
I am trying to run a sample model with the `example_config.yml` provided in the repo.

With a terminal opened in the root directory of the repo (/ctlearn), I ran the following two commands:
```

export CTLEARN_DIR=/home/jsevillamol/Documentos/ctlearn/ctlearn
python $CTLEARN_DIR/run_model.py config/example_config.yml
```

This produces the following error:

```
Traceback (most recent call last):
  File "/home/jsevillamol/Documentos/ctlearn/ctlearn/run_model.py", line 459, in <module>
    run_model(config, mode=args.mode, debug=args.debug, log_to_file=args.log_to_file)
  File "/home/jsevillamol/Documentos/ctlearn/ctlearn/run_model.py", line 65, in run_model
    model_module = importlib.import_module(config['Model']['model']['module'])
  File "/home/jsevillamol/anaconda3/envs/ctlearn/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'cnn_rnn'
```

But the `cnn_rnn.py` file is in its proper folder (`ctlearn/models`).

What am I doing wrong?

I am using Python 3.6 on Ubuntu 16.04, with Tensorflow-gpu 1.10

**Comment by Jsevillamol** [comment]
That was it! Looking forward to accessing some data so I can actually try these models for real.

**Comment by Jsevillamol** [comment]
Further problems:

1. When I try running `python $CTLEARN_DIR/run_model.py myconfig.yml` I get `ModuleNotFoundError: No module named 'ctlearn'`. Somehow the module ctlearn is not properly recognized.

2. When I try running a script such as this one (as suggested in the readme):

```
import yaml
from ctlearn.run_model import run_model

config = yaml.load('myconfig.yml')
run_model(config, mode='train', debug=True, log_to_file=True)
``` 
I get an error because the yaml file is not properly loaded. This can be fixed like this:

```
import yaml
from ctlearn.run_model import run_model

with open('myconfig.yml', 'r') as stream:
		config = yaml.load(stream)

run_model(config, mode='train', debug=True, log_to_file=True)
```

3. However, when running the script above I get `ModuleNotFoundError: No module named 'cnn_rnn'`.

I am using Python 3.6.

**Comment by Jsevillamol** [comment]
@aribrill thanks to you!

1. Turns out I had forgotten to run `pip install --upgrade .`. After running this the invocation from the command line works!
2. :+1: 
3. I wrote a typo on the model path. Ooops!

I think it is working now, though it is hard to say because running `python $CTLEARN_DIR/run_model.py my_config/example_config.yml` does not seem to produce any message on the console :stuck_out_tongue:
